app/javascript/components/components/Header.js

```diff
@@ -17,18 +17,32 @@ import { NavLink } from 'react-router-dom'
                 <p>This is the header</p>
                 <nav>
                     <ul>
-                        {logged_in && <a href="/surfspotindex">
-                            Surf Breaks
-                        </a>}
-                        {logged_in && <a href="/mycollectionsindex">
-                            My Collections
-                        </a>}
-                        {logged_in && <a href={sign_out_route}>
-                            Sign Out
-                        </a>}
+                        <a href="/">
```

Comment:
When creating internal navigation, you don't want to use href as it will force a page refresh. Devise routes can use href, but your components should not.

Reply:
Ohhh got it! Just fixed that.